pkg/agent/core/ngt/handler/grpc/index_test.go

```diff
@@ -62,6 +62,26 @@ func Test_server_CreateIndex(t *testing.T) {
 		}
 		return nil
 	}
+
+	/*
+		- Equivalence Class Testing
+			- case 1.1: success to create index with 1 uncommitted insert index
+			- case 1.2: success to create index with 100 uncommitted insert index
+			- case 2.1: success to create index with 1 uncommitted delete index
+			- case 2.2: success to create index with 100 uncommitted delete index
+			- case 3.1: success to create index with 1 uncommitted insert + delete index
+			- case 3.2: success to create index with 100 uncommitted insert + delete index
```

Comment:
how many delete index?

Reply:
100. Should I represent like this?
```suggestion
			- case 3.1: success to create index with 1 uncommitted insert & delete index
			- case 3.2: success to create index with 100 uncommitted insert & delete index
```

Reply:
yes plz